compiler/rustc_codegen_llvm/src/gotoc/mir_to_goto/context/goto_ctx.rs

```diff
@@ -70,10 +75,16 @@ impl<'tcx> GotocCtx<'tcx> {
 
 /// Getters
 impl<'tcx> GotocCtx<'tcx> {
-    pub fn crate_name(&self) -> String {
+    /// The short crate name without versioning information.
+    pub fn short_crate_name(&self) -> String {
         self.tcx.crate_name(LOCAL_CRATE).to_string()
     }
 
+    /// The full crate name including versioning info
+    pub fn full_crate_name(&self) -> String {
+        self.full_crate_name.clone()
```

Comment:
Do we want to clone here? Or should we return `&str`?

Reply:
Changed.